src/@batch-flask/ui/duration-picker/duration-picker.component.ts

```diff
@@ -76,7 +76,7 @@ export class DurationPickerComponent implements FormFieldControl<any>,
     @Input() public allowUnlimited: boolean = true;
     @Input() public defaultDuration: string = "";
 
-    @Input() @FlagInput() public required = false;
+    @Input() @FlagInput() public required: boolean | string = false;
```

Comment:
why should this be a string?

Reply:
the `@FlagInput` should also convert the value to `true` or `false` when you use it without a value i.e `<foo required>` instead of `<foo [required]="true">`

Reply:
Mainly to mirror https://angular.io/api/forms/RequiredValidator#properties. I'll change it back since we make use of flaginput